history-network.md

```diff
@@ -85,6 +94,13 @@ content-key  = Container(chain-id: uint16, content-type: uint8, block-hash: Byte
 content-type = 0x03
 ```
 
+#### Transactions
+
+```
+content-key = Container(chain-id: uint16, content-type: uint8, block-hash: Bytes32, transaction-hash: Bytes32)
```

Comment:
shouldn't `transaction-hash: Bytes32 ` actually be `transaction-index: uint32` ?

By having it as `transaction-hash` we force each implementation to keep index `transaction-hash -> transaction`  so it is addtional overhead, also there will be (from what I know) another overlay network `transaction index` which will keep data `transaction-hash -> (blockhash, index)` .

Also maybe simpler it would be define content-key as:
`Container(content-type: uint8, body-content-key: Container, transaction-index: uint32)` where
```
body-content-key = Container(chain-id: uint16, content-type: uint8 = 0x02, block-hash: Bytes32)
content-type = 0x04
```

And then `content-id` for it is just `H(body-content-key). It will ease up implementation, and mirrors api for getting transaction via json rpc : `eth_getTransactionByBlockHashAndIndex` which takes hash and index as arguments